src/app/admin/components/reports/reports.directive.js

```diff
@@ -398,6 +405,8 @@
                         var statusActivity = angular.copy(activity);
                         statusActivity.details = statusChange;
                         output.status.push(statusActivity);
+
+                        activity.details.push('<span class="bg-danger"><strong>' + statusChange + '</strong></span>');
                     }
                     if (data[i].newData.certificationEdition.name === '2011') {
                         activity.action = '<span class="bg-danger">' + activity.action + '</span>';
```

Comment:
Why is this marked as "bg-danger"? That means "questionable"

Reply:
Ah ok, I was confused, I thought you meant in your previous comment that this line should be added back in

Reply:
There were two basic things that needed to be removed for this ticket; the first was the actual "questionable activities" report, which would have entailed some navigation changes, a chunk of HTML deleted, and removing wherever the code was checking for questionable stuff to fill the list of activities.

The second part was un-highlighting questionable stuff wherever it showed up. Anything that was questionable before was highlighted in the various "other" Certified Product reports, and likely in the Version/Product reports too, and that highlight needed to be removed wherever it was found